frontend/uddhar-frontend/src/authentication/pages/Login.jsx

```
@@ -1,4 +1,125 @@
+              <div className="text-red-500 text-sm md:text-lg font-sans mt-2">
+                {error}
+              </div>
+            )}
+            <div className="flex flex-col gap-1 md:gap-5">
+              <button
+                type="submit"
+                className="w-2/3  self-center rounded-md bg-amber-700 text-white text-sm md:text-xl mt-2 py-2"
+              >
+                Login
+              </button>
+              <button className="w-full text-blue-400 font-bold text-sm md:text-lg ">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)